### DISCUSSION TITLE
无法打开

### DISCUSSION THREAD

**Discussion by 0xffffharry** [HMBSbige/NatTypeTester#205] [Q&A] [answered]
下载是最新的 [NatTypeTester-6.2.0.7z](https://github.com/HMBSbige/NatTypeTester/releases/download/6.2.0/NatTypeTester-6.2.0.7z) 环境win11，无法打开，需要安装什么依赖吗

**Comment by HMBSbige** [comment, marked as answer]
.NET Runtime 6.0

正常来说会弹窗提示安装
